modules/core/server/views/layout.server.view.html

```diff
@@ -45,7 +45,17 @@
 
   <!--Embedding The User Object-->
   <script type="text/javascript">
-    var user = {{ user | json | safe }};
+    var user = {
+      displayName: "{{ user.displayName | escape }}",
```

Comment:
This part of the express middleware is still a bit *elusive* to me; meaning I don't understand it that well..

This is how I'm perceiving it to be working, in regards to the Authentication service. When this object is set on the browser, it becomes accessible through `$window.user`. Is that correct?

With the above assumption..

I think we can just set this variable to an empty object. However, how would we do this? In the current state of this PR, if the user isn't authenticated then the express variable is undefined. Thus, this variable is set to the following..

```javascript
{
  displayName: "",
  provider: "",
  username: "",
  created: "",
  roles: "",
  profileImageURL: "",
  email: "",
  lastName: "",
  firstName: ""
}
```

This is probably fine. However, we need to adjust the logic in the Authentication service. At this line https://github.com/meanjs/mean/blob/master/modules/users/client/services/authentication.client.service.js#L7, we probably can change it to..
`user: $window.user.username.length ? $window.user : {}`, or something similar